doc/usecases/plugins/spec_globbing.md

```diff
@@ -0,0 +1,51 @@
+# Use Case: Specification for application communicating with multiple services
```

Comment:
This is too specific for a usecase.
About the filename: "globbing" should be introduced as term, if you really want it. The use case should be called like the file name. Be consistent with your terms.